app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -52,9 +52,16 @@ def find_vm_by_uuid(uuid)
 
     def create_vm(args = { })
       args = vm_instance_defaults.merge(args.to_h.symbolize_keys).deep_symbolize_keys
+      # Merge AWS EC2 tags
+      new_tags = {}
       if (name = args[:name])
-        args[:tags] = {:Name => name}
+        new_tags = {:Name => name}
       end
+      args[:tags]&.each_line do |line|
+        key, value = line.strip.split(/=/)
```

Comment:
I believe we should add some form of validation if the user inputs anything we do not expect, i.e.:
```suggestion
        next unless line =~ /w+=w+/ 
```